torch/csrc/jit/runtime/symbolic_script.cpp

```diff
@@ -1388,8 +1388,9 @@ const std::vector<std::string> functions = {
                   mat2,
                   *,
                   beta: number,
-                  alpha: number):
-            result = torch.addmm(self, mat1, mat2, beta=beta, alpha=alpha)
+                  alpha: number,
+                  dtype: Optional[int]):
+            result = torch.addmm(self, mat1, mat2, beta=beta, alpha=alpha, dtype=dtype)
```

Comment:
below on line 1399 in `backward(grad_output)` return an extra `None`, e.g.
```
return self_grad, mat1_grad, mat2_grad, None, None
```
should change to 
```
return self_grad, mat1_grad, mat2_grad, None, None, None
```